[Shader Language] floatBitsToUInt() does not work

**Godot version:**
`Godot_v3.0.1-stable_win64.exe`


**OS/device including version:**
Windows 10


**Issue description:**
`uint test = floatBitsToUInt(1.0);` gives a 400+ line error in the console. Here's the main part:

```
ERROR: _display_error_with_code: CanvasShaderGLES3: Fragment Program Compilation Failed:
0(236) : error C1008: undefined variable "floatBitsToUInt"

   At: drivers\gles3\shader_gles3.cpp:168
ERROR: ShaderGLES3::get_current_version: CanvasShaderGLES3: Fragment Program Compilation Failed:
0(236) : error C1008: undefined variable "floatBitsToUInt"

   At: drivers\gles3\shader_gles3.cpp:441
```

--- a/servers/visual/shader_language.cpp
+++ b/servers/visual/shader_language.cpp
@@ -1641,10 +1641,10 @@ const ShaderLanguage::BuiltinFuncDef ShaderLanguage::builtin_func_defs[] = {
 	{ "floatBitsToInt", TYPE_IVEC3, { TYPE_VEC3, TYPE_VOID } },
 	{ "floatBitsToInt", TYPE_IVEC4, { TYPE_VEC4, TYPE_VOID } },
 
-	{ "floatBitsToUInt", TYPE_UINT, { TYPE_FLOAT, TYPE_VOID } },
-	{ "floatBitsToUInt", TYPE_UVEC2, { TYPE_VEC2, TYPE_VOID } },
-	{ "floatBitsToUInt", TYPE_UVEC3, { TYPE_VEC3, TYPE_VOID } },
-	{ "floatBitsToUInt", TYPE_UVEC4, { TYPE_VEC4, TYPE_VOID } },
+	{ "floatBitsToUint", TYPE_UINT, { TYPE_FLOAT, TYPE_VOID } },
+	{ "floatBitsToUint", TYPE_UVEC2, { TYPE_VEC2, TYPE_VOID } },
+	{ "floatBitsToUint", TYPE_UVEC3, { TYPE_VEC3, TYPE_VOID } },
+	{ "floatBitsToUint", TYPE_UVEC4, { TYPE_VEC4, TYPE_VOID } },
 
 	{ "intBitsToFloat", TYPE_FLOAT, { TYPE_INT, TYPE_VOID } },
 	{ "intBitsToFloat", TYPE_VEC2, { TYPE_IVEC2, TYPE_VOID } },

